modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div 
+    v-if="guest"
+    class="admin-bar-guest"
+  >
+    <button @click="logout">Log Out</button>
+  </div>
+  <div v-else
```

Comment:
The HTML from here down is unmodified. There are a few changes separately called out in the `script` section.